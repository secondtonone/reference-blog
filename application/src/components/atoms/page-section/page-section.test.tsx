/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import PageSection from '.';

describe('Match snapshot PageSection', () => {
  it('empty', () => {
    const { asFragment } = render(<PageSection />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('with children', () => {
    const { asFragment } = render(<PageSection><div className="test" /></PageSection>);

    expect(asFragment()).toMatchSnapshot();
  });
});
