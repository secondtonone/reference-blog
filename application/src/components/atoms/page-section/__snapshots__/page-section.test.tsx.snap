// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshot PageSection empty 1`] = `
<DocumentFragment>
  <div
    class="page-section__SectionStyled-sc-rosswz-0 byJLpF"
  />
</DocumentFragment>
`;

exports[`Match snapshot PageSection with children 1`] = `
<DocumentFragment>
  <div
    class="page-section__SectionStyled-sc-rosswz-0 byJLpF"
  >
    <div
      class="test"
    />
  </div>
</DocumentFragment>
`;
